code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -18,7 +18,7 @@ insert ascii eagle on american flag background here
 	container_type = OPENCONTAINER
 	var/obj/item/frying = null	//What's being fried RIGHT NOW?
 	var/cook_time = 0
-
+	var/list/blacklisted_items
```

Comment:
1. it must be ```/static``` I demand it
2. making it ```/static``` means you can move ```... = typecacheof(...)``` into the definition here, which is better as it'll only run once vs n-times